---
title: "Logboek Mirte"
author: "Mirte Draaijer"
date: "`r Sys.Date()`"
bibliography: "logfile_mirte.bib"
output:
  html_document:
    toc: yes
    toc_depth: 3
    toc_float:
      collapsed: yes
      smooth_scroll: yes
lang: "en"
editor_options: 
  markdown: 
    wrap: sentence
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r, include = FALSE}
#Loading libraries:
library("readxl")
library("DESeq2")
library("S4Vectors")
library("EnhancedVolcano")
```


# Introduction:

In this log I will work on a multi-omics project for a school subject called "Omics". During this project I am working in a group with Ivar, Jasper, and Kasthury. In this log I will only describe the steps I took during the project, but I will reference their logs where necessary. 
The data we will be using can be downloaded through the following links:

- [RNA sequencing](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE271272)
- [Proteomics](https://www.ebi.ac.uk/pride/archive/projects/PXD053766)

The corresponding article can be found [here](https://www.nature.com/articles/s41536-025-00404-3).

The question we want to answer during this project is: _Which proteins are significantly expressed in the EVs and SFs and does that influence the transcriptome of the control group?_ 

# Gathering information:

## Reading the article:

**23-4-2025 & 24-4-2025:**

For this project we will be using data from the article "Fibroblast-derived osteoglycin promotes epithelial cell repair", as a first step in this project I will read through the article and shortly summarize / make notes about the important parts.

**Summary / article notes:**

<span style="text-decoration:underline">**Introduction:**</span>
In this study the researchers aimed to identify factors in the lung fibroblast secretome with regenaritve potential for therapeutic development. They did this using a proteomics-guided drug discovery strategy.

<span style="text-decoration:underline">**Results:**</span>
The following quotes sumarize the results:

- "The CFE was significantly increased by treating cultures with EVs (109 particles/mL, 50,000 particles per cell) or SFs (30µg/mL) (Fig. 1a–c), without affecting organoid size (Fig. 1d). In addition, EV- or SF-treatment increased organoid formation in a concentration-dependent manner (Fig. S2a–d). Interestingly, reducing the number of fibroblasts yielded significantly fewer murine organoids, underscoring the supportive function of fibroblasts in epithelial organoid formation (Fig. S2e–f)."
- "In summary, the proteomic analysis collectively demonstrates an enrichment of EV-associated proteins in EVs when compared to SFs."
- "In total, we identified twelve factors for evaluation in lung organoids: seven factors within the SF-protein group and five factors present in both sample types (Fig. 2a, b, Supplementary Data 3)."
- "This suggests that fibroblasts, known to secrete important components of the extracellular matrix, such as collagens, are a likely endogenous source for OGN."
- "This suggests that while the overall area of OGN-positive tissue may be reduced, the remaining cells within these regions exhibit stronger OGN expression, potentially reflecting an active repair response in surviving cells27. Taken together, these data suggest that OGN expression is mainly affected by current smoking."
- "Taken together, these data demonstrate the profound potential of OGN on the human and mouse alveolar epithelium, positioning it as a promising therapeutic candidate supporting alveolar epithelial repair."
- "Overall, EVs, SFs, and OGN had the most pronounced transcriptomic effect in fibroblasts compared to epithelial cells."
- "Together, these findings suggest enhanced epithelial-fibroblast crosstalk mediated by growth factors upon OGN treatment, potentially contributing to the increased organoid formation."
- "In summary, these results indicate that a smaller active fragment of OGN is sufficient to support alveolar epithelial progenitor behavior."
- "Taken together, the active fragment of OGN ameliorates elastase-induced lung injury in PCLS and improves lung function in mice, thereby underscoring its potential as a promising therapeutic approach."

These quotes will make it easier to look up certain results when I need to know more about them.

<span style="text-decoration:underline">**Discussion:**</span>
The discussion talks about how to interpret the results, besides this it describes limitation of the study as well. The most important limitations in the study are:

- Relatively  small number of patients.
- The use of human lung fibroblast-derived EVs and SFs in a murine organoid system.

<span style="text-decoration:underline">**Methods:**</span>
For our project the bio-informatics methods are most interesting, so I will only make notes one these. If needed I will summarize the other methods later on as well.

<span style="text-decoration:underline">Bulk RNA sequencing analysis:</span>

- Total RNA was extracted from cells resorted from organoids using the NucleoSpin RNA isolation kit according to the manufacturer's instructions.
- Bulk RNA sequencing on resorted Epcam+ and fibroblasts from organoids was performed by GenomeSCan using an Illumina NovaSeq 6000 sequencer. 
- FastQC v0.34 and FastQA were used to preform quality control.
- Trimmomatic v0.30 was used to remove adapter sequences from the reads.
- The reads were aligned against the ensemble mouse reference GRCm38.
- Principal component analyses were preformed in R using DESeq2.
- Differentially expressed genes between control and treated samples were found using DESeq2.
- Kyoto encyclopedia of genes and genomes was used as reference database.

<span style="text-decoration:underline">Proteomic analysis:</span>

- Liquid chromatography-tandem mass spectrometry data was processed with Spectronaut using the standard settings of the directDIA workflow with a human SwissProt database.
- For quantification local normalization was applied, and Q-value filtering was adjusted to the classical setting without imputing.

<span style="text-decoration:underline">Statistical analysis:</span>

- Results are shown as mean ± standard error, with sample size and repeats in figure legends.
- Four normality tests (D'Agostino-Pearson, Anderson-Darling, Shapiro-Wilk, Kolmogorov-Smirnov) were conducted.
- One-way ANOVA tests were applied
- Unpaired/paired two-tailed Student's t test or Mann-Whitney and Wilcoxon tests were applied for two group comparisons.
- Analyses were done in GraphPad Prism 10 and RNA sequencing analysis in R Studio.

## Literature assignment:

**23-4-2025:**

For this subject we have a literature assignment, we have to look up 2 research articles related to our project and summarize them. I will document this in my log as well, so that I have a nice overview of all my information. We will look up articles on the following things:

- COPD: Jasper will look up 2 research articles about COPD, this is the example the researchers used in the article that we're using for our project.
- Epithelial cell repair: Ivar will look up 2 research articles about epithelial cell repair.
- Multi-omics analysis: I will look up 2 articles about multi-omics analysis, we can use the summary of these articles as a guideline for our own project.
- Proteomics: Kasthury will look up 2 articles about proteomics, we have never worked with proteomics before. Having some more background information on this subject will come in useful. 

**24-4-2025:**

Today I will start looking up some potential articles about multi-omics analysis. These are titles of the potential articles I have found:

- "Bioinformatics Databases and Tools for Analysis of Multi-omics"
- "Multi-omics approaches to disease"

I will tell my group that I have found 2 articles for my part of this assignment and aks them if I can continue or if they need to check the articles first. I checked with my teammates and they think the articles I found are good, so I will be using these for the literature assignment.

### Bioinformatics Databases and Tools for Analysis of Multi-omics:

**24-4-2025 & 29-4-2025:**

On these days I worked on summarizing this article for the literature assignment.

### Multi-omics approaches to diseases:

**24-4-2025 & 6-5-2025:**

On these days I worked on summarizing this article for the literature assignment.

## Guest lecture:

**8-2-2025:**

Today we had a guest lecture from Karin Wolters.

She is one of the researchers that worked on the project we chose. So we had the opportunity to ask her some questions.

Our most important question was about the data, we could not really make up which samples from the files match up. This makes it hard to compare the results when we are going to start the data analysis. Karin said that she will ask a colleague about this, and that our teachers will get back to us with the information needed.

# Problem with the data:

**13-5-2025 & 14-5-2025:**

Previously I described that we had a guest lecture from Karin Wolters, we asked her some questions about our data. Today we got a response from Karin. 

The EVs used for the transcriptomics dataset and the EVs used for the proteomics dataset come from completely different batches. This is a problem because this way we can not compare the two. For now our plan is to discuss with our teacher what to do next, because with these datasets we can not preform a multi-omics project. Most likely we will have to use a different dataset or change our entire project.

We discussed the problem with our teacher and decided to change our research question, luckily we can use our initial approach for now. In the later steps we might need to make some changes.

# Research question:

**8-5-2025:**

As a group we decided that everyone will think of a research question for this project, we settled on a deadline of next monday. 

**11-5-2025:**

The research question I came up with is the following _What genes and proteins are differentially expressed in the control- and researchgroups in the dataset?_ since we need to write the article in Dutch I translated it as well: _Welke genen en eiwitten zijn differentieel tot expressie gebracht tussen de controle- en onderzoeksgroepen in de dataset?_

I also thought about which tools we can use to answer the research question I came up with. This is what I have so far:

- DESeq2 for the differential gene expression analysis, we have used this tool before which will make it an easy choice for this portion of the project.
- For the differential protein expression there are multiple tools/libraries available some examples are: limma, MSstats, DEP, and prolfqua.
- We can visualize the results as venn-diagrams and vulcano plots.
- Furthermore we could take a look at KEGG pathways to see if there are any pathways that stand out.

**12-5-2025:**

Today we discussed which research question we will use for this project. Ofcourse this is something that can change later on but for now we settled on: _Are there any pathways that changed in the COPD researchgroup in comparison to  the control group?_ and in Dutch: _Zijn er pathways veranderd in de COPD onderzoeksgroep ten opzichte van de controlegroep?_

**14-5-2025:**

As I discussed in the chapter ```Problem with the data```, we decided to change our research question based on the problem. Our new research question is: _Which proteins are significantly expressed in the EVs and SFs and does that influence the transcriptome of the control group?_ and in Dutch: _Welke eiwitten zijn in de EV’s en de SF’s significant aanwezig en heeft dat invloed op het transcriptoom van de controle, EV’s en SF’s?_

# Project approach:

**12-5-2025:**

Now that we have settled on a research question, it is important to discuss our approach for this project. What steps will we take? How will we preform these steps? Who will preform them? Below is a small overview of our initial approach.

| Task                             | Person   |
|----------------------------------|----------|
| DESeq                            | Mirte    |
| DESeq alternative proteomics     | Ivar     |
| Vulcano plot transcriptomics     | Mirte    |
| Vulcano plot proteomics          | Ivar     |
| Pathway analysis transcriptomics | Jasper   |
| Pathway analysis proteomics      | Kasthury |

This will be our initial approach for the project, ofcourse if we run into new insights or problems (hopefully not) we will change the approach. But for now this is how we will start. 

# DESeq:

## Proteomics:

Ivar is going to use DESeq for analysis of the proteomics data.

## Transcriptomics:

**12-5-2025:**

During a meeting with my group we discussed that I will work on DESeq for differential gene expression. Karin still needs to get back to us about which sample is which. But I can start on writing code for implementing DESeq and I can test on data I still have from an old project. 

### Background info:

**12-5-2025:**

DESeq2 is used for detecting differentially expressed genes from count data.  The package DESeq2 provides methods to test for differential expression by use of negative binomial generalized linear models. 

[@love-2014]

### Implementing:

**12-5-2025:**

Karin still needs to get back to us on which sample is which, for now I will implement DESeq to use all samples, but I can change this later on if we're not going to use all samples. If I have working code changing it to use the correct groups will be easy.

#### Step 1 loading the data:

```{r}
rna_seq_data <- read.csv(
  "omic_epithelial_cell_repair/data/GSE271272_rawcounts.csv")

head(rna_seq_data)
```

When taking a look at the first few lines of the RNA sequencing data I'm seeing the following things:

- The first column is called ```Sample```, I looked up the first few lines of this row and discovered that these are genes.
- The following columns are named in the following way: starting with M1, M2, or M3. Followed by ctrl, EVs, SFs, or OGN. Followed by Epcam or CCL. When taking a  look at the article I found that SF stand for soluble factors, EV stands for extracellular vesicles, OGN stands for osteoglycin. I think M1, M2, and M3 indicate the duplicates. I think Epcam and CCL indicate which cell type was used.
- Each column except the first contains a number, I think this number is a count representing how many reads of the gene were found in the sample.

#### Step 2 constructing DESeq dataset:

To construct the DESeq dataset, the original data needs to be reconstructed. 

```{r}
cts <- data.frame(rna_seq_data[,-1])
colnames(cts) <- colnames(rna_seq_data)[-1]
row.names(cts) <- rna_seq_data$Sample
cts <- as.matrix(cts)
```

After preforming this step I can see that it went well. Now it is time to construct a metadata dataframe. This metadata dataframe contains the sample name, group, and cell type.

```{r}
sample_names <- c("M1_ctrl_Epcam", "M2_ctrl_Epcam", "M3_ctrl_Epcam", "M4_ctrl_Epcam",
            "M1_EVs_Epcam", "M2_EVs_Epcam", "M3_EVs_Epcam", "M4_EVs_Epcam",
            "M1_SFs_Epcam", "M2_SFs_Epcam", "M3_SFs_Epcam", "M4_SFs_Epcam",
            "M1_OGN_Epcam", "M2_OGN_Epcam", "M3_OGN_Epcam", "M4_OGN_Epcam",
            "M1_ctrl_CCL", "M2_ctrl_CCL", "M3_ctrl_CCL", "M4_ctrl_CCL",
            "M1_EVs_CCL", "M2_EVs_CCL", "M3_EVs_CCL", "M4_EVs_CCL",
            "M1_SFs_CCL", "M2_SFs_CCL", "M3_SFs_CCL", "M4_SFs_CCL",
            "M1_OGN_CCL", "M2_OGN_CCL", "M3_OGN_CCL", "M4_OGN_CCL")

group <- rep(c("control", "extracellular_vesicles", "soluble_factors", "osteoglycin"), each = 4, times = 2)
type <- rep(c("epcam", "ccl"), each = 16)

metadata <- data.frame(group = factor(group), type = type)
rownames(metadata) <- sample_names

head(metadata)
```

Looking at the metadata dataframe, I can see that the creation of this dataframe went well. Now it is time to construct the DESeqdataset (dds).

```{r}
dds <- DESeqDataSetFromMatrix(countData = cts,
                              colData = metadata,
                              design = ~ group)
```

**13-5-2025:**

Today I wanted to continue working on DESeq, I reran all of my code chunks but I ran into a problem with the one above. I keep getting the error/warning:
```
Warning: some variables in design formula are characters, converting to factorsError in validObject(.Object) :
  invalid class “DESeqDataSet” object: superclass "ExpData" not defined in the environment of the object's class
```
This is a problem because now I can not continue, I googled the error but can't find much information. I asked my teacher for help but he does not have any experience with DESeq, so he contacted another teacher. Hopefully the other teacher can help me.

The other teacher said it might be a problem with DESeq, so he suggested to try it on the R studio server. I tried running the code on our R studio server and this fixed the problem. So for now I will continue working on the server.

**14-5-2025:**

Today I deleted my installation of DESeq2 and reinstalled it, this suggestion was made by the teacher who helped me sort of fix the issue with DESeq2. Deleting and reinstalling DESeq2 made it work again.

#### Step 3 generating summary:

As a first step to analyse the data I will create a summary for each group combination:

- Control vs extracellular vesicles
- Control vs soluble factors
- Extracellular vesicles vs soluble factors

```{r}
dds <- DESeq(dds)
control_vs_extracellular <- results(dds, contrast = c("group", "control", "extracellular_vesicles"))
control_vs_soluble <- results(dds, contrast = c("group", "control", "soluble_factors"))
extracellular_vs_soluble <- results(dds, contrast = c("group", "extracellular_vesicles", "soluble_factors"))
```

```{r}
summary(control_vs_extracellular)
```
When looking at the results of the summary control vs extracellular the following things stand out:

- 13 genes are upregulated.
- 72 genes are downregulated.
- There are 5041 low counts.


```{r}
summary(control_vs_soluble)
```

When looking at the results of the summary control vs soluble the following things stand out:

- 0 genes are upregulated.
- 3 genes are downregulated.

```{r}
summary(extracellular_vs_soluble)
```

When looking at the results of the summary extracellular vs soluble the following things stand out:

- 88 genes are upregulated.
- 17 genes are downregulated.
- There are 5728 low counts.

Over all it stands out that there is quite a big portion of low counts varying from 43% (control vs extracellular) to 48% (extracellular vs soluble).

#### Step 4 PCA:

**14-5-2025:**

After creating the summaries I can preform PCA and create plots of it. First I will preform PCA based on cell type (CCL and epcam)

```{r, message = FALSE}
# Log2 transformation and centering data:
dds <- estimateSizeFactors(dds)
se <- SummarizedExperiment(log2(counts(dds, normalized = TRUE) + 1),
                           colData = colData(dds))

plotPCA(DESeqTransform(se), intgroup = "type",
        pcsToUse = 1:2,
        ntop = 11818)
```

Looking at the output of the PCA I can clearly see that the groups cluster. The epcam cells are all the way at the left and the CCL cells are all the way to the right. Since the samples are from different cell types this is not a suprise. 

Now I will preform a PCA based on group (control, extracellular vesicles, osteoglycin, and soluble factors).
 
```{r, message = FALSE}
plotPCA(DESeqTransform(se), intgroup = "group",
        pcsToUse = 1:2,
        ntop = 11818)
```

When looking at the output of the PCA I can clearly see that approximately half of the samples are clustered on the left and the other half are clustered on the right, this can be explained by the cell types like I discussed in the previous PCA. 
On the right side the CCL cells are shown, here I can see that the extracellular vesicles cluster slightly. Distinguishing between the other groups is difficult in this PCA, I can do it because of the colors, but based on the clustering distinguishing them would be hard. 

Most of the variance is shown on PC1 (73%), when looking at the plot this makes sense because on PC1 the different cell types (CCL and epcam) are seperated. 8% of the variance is explained by PC2 this is the variance between the different groups (control, extracellular vesicles, osteoglycin, and soluble factors).

When comparing it to the PCA Jasper preformed there are differences. I Discussed this with my teacher, and this is most likely caused by how DESeq works. An interesting thing to further explore is preforming PCA for each cell type individually. 

##### Comparison to original PCA:

**15-5-2025:**

Yesterday both Jasper and I preformed PCA, we had different results. So I decided to take a closer look at the transcriptomics analysis of the original article. 

In figure 1b you can see the PCA preformed in the article from which we are using the data. Yesterday I worked on preforming PCA, I got a simmiliar result. I decided not to use this result because it did not use all genes and used as variance stabilizing normalization, and I used a log2 transformation. However Since they used this in the original article, I am going to preform PCA this way as well.

```{r, echo=FALSE, fig.cap="Figure 1: An image containing 11 different plots about transcriptomics. The image was taken from the original article.", out.width="75%", fig.pos="H"}
    knitr::include_graphics("transcriptomics_original.png")
```

Below you can see the new PCA preformed in (presumably) the same way as in the article. 

```{r, message = FALSE}
vsd <- vst(dds, nsub=500)

plotPCA(vsd, intgroup = "group",
        pcsToUse = 1:2,
        ntop = 500) +
  aes(shape = type) +
  scale_shape_manual(values = c("ccl" = 19, "epcam" = 17))
```

When looking at the new PCA I can see that it is almost identical to the one from the original article (figure 1b), however there are some differences:

- The scale of the X-axis is different, but when looking at the position of the data points in comparison to the original the values look the same.
- My plot uses a different theme than the original but that is a personal choice.
- My plot uses different sample names, but that is a personal choice as well.
- In my plot I used different shapes to indicate the cell type, in the original they used different colors to indicate the cell type. I think my version is better because this way you can compare the different groups of the different cell types easier than when they are different colors.

Besides comparing my PCA to the original it is important to interpret it as well. PC1 explains 95% of the variance, this seems to be caused by the different cell types (CCL and epcam). PC2 explains 2% of the variance. When looking at the plot there is some clustering of the different groups but some are hard to distinguish, mainly the soluble factors and the extracellular vesicles. 
Since so much of the variance is caused by the different cell types, it would be interesting to look at the two cell types separately.

# Volcano plots:

## Proteomics:

Ivar is going to make volcano plots for the proteomics data.

## Transcriptomics:

**15-5-2025:**

In the original article they made volcano plots as well (figure 1c and figure 1e). In these volcano plots they compared the response to OGN versus control in epithelial progenitor cells (figure 1c) and they compared the response to OGN versus versus control in fibroblasts (figure 1e). 
For our research question we want to compare the EVs and the SFs, so OGN is not of interest to us. For this reason I will not be replicating the original volcano plots, but create volcano plots using other groups: extracellular vesicles versus soluble factors, control versus extracellular vesicles, and control versus soluble factors. 

A volcano plot is a type of scatter plot used for differntial expression analysis. On the top left it shows genes that are downregulated, on the top right it shows genes that are upregulated, and on the bottom center is shows genes with little change.

I will start by making a volcano plot voor the extracellular vesicles versus the soluble factors.

```{r, fig.height=5.5}
  EnhancedVolcano(extracellular_vs_soluble,
    lab = rownames(extracellular_vs_soluble),
    x = "log2FoldChange",
    y = "pvalue",
    title = "Extracellular vesicles vs. soluble factors")
```

Interpretation extracellular vesicles vs. soluble factors:

- The green dots show genes that passed that fold change threshold but not the p-value threshold.
- The blue dots show genes that passed the p-value threshold but not the fold change threshold.
- The grey dots show genes that passed neither thresholds.

```{r, fig.height = 5.5}
  EnhancedVolcano(control_vs_extracellular,
    lab = rownames(control_vs_extracellular),
    x = 'log2FoldChange',
    y = 'pvalue',
    title = "control vs. extracellular vesicles")
```

Interpretation control vs. extracellular vesicles:

- The red dots show genes that are both significant by p-value and fold change. These genes are most likely very meaningful. The genes that are red in the plot are: Saa3, Lcn2, and Slpi all of these genes are downregulated.
- The green dots show genes that passed that fold change threshold but not the p-value threshold.
- The blue dots show genes that passed the p-value threshold but not the fold change threshold.
- The grey dots show genes that passed neither thresholds.

```{r, fig.height = 5.5}
  EnhancedVolcano(control_vs_soluble,
    lab = rownames(control_vs_soluble),
    x = 'log2FoldChange',
    y = 'pvalue',
    title = "control vs. soluble factors")
```

Interpretation control vs. soluble factors:

- The red dots show genes that are both significant by p-value and fold change. These genes are most likely very meaningful. The genes that are red in the plot are: Saa3, and Lcn2 all of these genes are downregulated.
- The green dots show genes that passed that fold change threshold but not the p-value threshold.
- The grey dots show genes that passed neither thresholds.

I think it would be useful to look more into the following genes for our project: Saa3, Lcn2, and Slpi. For further analysis we will preform pathway analysis as well.

# Pathway analysis:

## Proteomics:

Kasthury is going to preform pathway analysis for the proteomics.

## Transcriptomics:

Jasper is going to preform pathway analysis for the transcriptomics data.

# PLS-DA:

For finding the connections between the two datasets we will be preforming PLS-DA. As a first step I will look up some more information about PLS-DA, what is it? How can we use it?

## Gathering information:

**19-5-2025:**

PLS-DA was recommended to us by our teacher for this project. I will look up some more information about it, so that we can use it in our project.
Partial least squares-discriminant analysis (PLS-DA) is a versatile algorithm that can be used for predictive and descriptive modellings as well for discriminative variable selection.

[@lee-2018]

PLS-DA is often used for identifying biomarkers, distinguishing between physiological states, and predicting class membership for new samples based on their metabolite profiles.

[@metabolon-inc-2025]

Since our research question is: _Which proteins are significantly expressed in the EVs and SFs and does that influence the transcriptome of the control group?_ PLS-DA will be a good contestant for analyzing the data. Tomorrow We will ask our teacher for some more help on implementing PLS-DA.

As a group we talked about implementing PLS-DA, for the information we have now on PLS-DA this is how we want to preform it: 

- PLS-DA individually on transcriptomics data and individually on proteomics data.
- PLS-DA across the two datastes.

The individual implementation is something we can already start working on, PLS-DA across the two datasets is something we need to gather some more information on we will do this by asking our teacher.

# VSD:







**EVEN KIJKEN HOE IK DIT DEEL IN MIJN LOGBOEK WIL DOEN!!!**

# Exploratory data analysis:

## Data loading and first look:

**24-4-2025:**

The first step in the exploratory data analysis is loading the data.

```{r}
proteomics_data <- read_excel(
  "omic_epithelial_cell_repair/data/2021-36_2021-42_SN16withHELAlib_MS2Quant_Qclassic_LocalNorm_NoImp.xlsx")

rna_seq_data <- read.csv(
  "omic_epithelial_cell_repair/data/GSE271272_rawcounts.csv")
```

Now I will take a look at the first few lines of both files, starting with the proteomics data.

```{r}
head(proteomics_data)
```

When taking a look at the first few lines of the proteomics data I'm seeing the following things:

- The first column is called ```PG.ProteinGroups```this column contains one or more protein groups.
- The second column is called ```PG.Genes``` the first row of this column contains the word ```COUNT```, when looking at the other number it shows a number. I'm assuming that this is a count, I'm not yet sure what this count means. After the first row this column contains one or more genes.
- The next columns have a name consisting of what seems to be a year, either the number 36 or 42, SF or EV followed by the number 1, 2, or 3. When taking a look at the article I found that SF stand for soluble factors and EV stands for extracellular vesicles. I'm not sure what the numbers 36 and 42 mean, I can't find anything about this. I think the numbers 1, 2, and 3 stand for duplicates.

Now I will take a look at the first few lines of the RNA sequencing data file.

```{r}
head(rna_seq_data)
```

When taking a look at the first few lines of the RNA sequencing data I'm seeing the following things:

- The first column is called ```Sample```, I looked up the first few lines of this row and discovered that these are genes.
- The following columns are named in the following way: starting with M1, M2, or M3. Followed by ctrl, EVs, SFs, or OGN. Followed by Epcam or CCL. When taking a  look at the article I found that SF stand for soluble factors, EV stands for extracellular vesicles, OGN stands for osteoglycin. I think M1, M2, and M3 indicate the duplicates. I think Epcam and CCL indicate which cell type was used.
- Each column except the first contains a number, I think this number is a count representing how many reads of the gene were found in the sample.


Now that I've looked at the first few lines it's important to compare the two files because we want to do analysis with both the files.

When looking at both the files it is hard to see which line in the proteomics data belongs to which line in the RNA sequencing data, because the samples do not have the same name. Another problem is the fact that the RNA sequencing data seems to have more samples. This is a problem, I think we need to discuss with our teachers.

# Comfirmatory data analysis